Rext/Rext.csproj

```diff
@@ -24,7 +24,7 @@ Safely handles request with JSON, XML and Form Data.</Description>
   </PropertyGroup>
 
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
-    <DocumentationFile>D:\DEV PROJECTS\WEB\Rextclient\Rext\RextHttpClient.xml</DocumentationFile>
+    <DocumentationFile>C:\Users\otega.ovie\Projects\Playground\RextHttpClient\Rext\RextHttpClient.xml</DocumentationFile>
```

Comment:
It would be better to use set relative path instead of absolute path

Reply:
That is for the Swagger. It always require an absolute path... When you pull the project all you need to do is change at the swagger property, it will update to yours. Its auto created

Reply:
Not Swagger, it's for the documentation you added via the top of methods and properties. It's the same file Swagger feeds on as well. And i don't think it can be set as relative. Changing it when pulling the project doesn't really take much. Thanks by the way!

Reply:
Ok. will have to look as wen I pull at first, it was looking for the xml doc file, that was specified at your path. I had to change. will confirm shall. thanks